hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -360,10 +360,10 @@ class Batch:
     def __init__(self,
                  client: 'BatchClient',
                  id: int,
-                 attributes: Dict[str, str],
+                 attributes: Optional[Dict[str, str]],
                  token: str,
                  *,
-                 last_known_status: Optional[bool] = None,
+                 last_known_status: Optional[Dict[str, Any]] = None,
```

Comment:
This one just had the wrong type. I didn't see any places where we sent it a bool.